js/index.js

```diff
@@ -1,3 +1,256 @@
+    </p>
+    <ul class="lags-used">
+      <li class="lag-used"><a class="a" href="#"></a></li>
+      <li class="lag-used"><a class="a" href="#"></a></li>
+      <li class="lag-used"><a class="a" href="#"></a></li>
+      <li class="lag-used"><a class="a" href="#"></a></li>
+    </ul>
+    <button class="see-project sp">See Project</button>
+  </div>
+</div>
+`;
+
+const workSectionSecCardOne = `
+<div class="project image-1">
+  <div class="overlay">
+    <div class="project-content">
+      <h2></h2>
+      <p></p>
+      <ul class="lags-used2">
+        <li><a class="a" href="#"></a></li>
+        <li><a class="a" href="#"></a></li>
+        <li><a class="a" href="#"></a></li>
+      </ul>
+    </div>
+  </div>
+  <button class="sp">See Project</button>
+</div>`;
+const workSectionSecCardTwo = `
+<div class="project image-2">
+  <div class="overlay">
+    <div class="project-content">
+      <h2></h2>
+      <p></p>
+      <ul class="lags-used2">
+        <li><a class="a" href="#"></a></li>
+        <li><a class="a" href="#"></a></li>
+        <li><a class="a" href="#"></a></li>
+      </ul>
+    </div>
+  </div>
+  <button class="sp">See Project</button>
+</div>`;
+const workSectionSecCardThr = `
+<div class="project image-3">
+  <div class="overlay">
+    <div class="project-content">
+      <h2></h2>
+      <p></p>
+      <ul class="lags-used2">
+        <li><a class="a" href="#"></a></li>
+        <li><a class="a" href="#"></a></li>
+        <li><a class="a" href="#"></a></li>
+      </ul>
+    </div>
+  </div>
+  <button class="sp">See Project</button>
+</div>`;
+
+window.addEventListener('load', () => {
+  const section = document.createElement('section');
+  section.setAttribute('id', 'portfolio');
+  section.classList.add('recent-work');
+
+  section.innerHTML = workSection;
+  document.querySelector('.headline').after(section);
+  document.querySelector('.main-project').innerHTML = workSectionMainCard;
+  document.querySelector('.secondary-projects').innerHTML = workSectionSecCardOne
+    + workSectionSecCardTwo
+    + workSectionSecCardThr
+    + workSectionSecCardOne
+    + workSectionSecCardTwo
+    + workSectionSecCardThr;
+  //
+  const mainCard = document.querySelector('.main-article');
+  mainCard.querySelector('img').setAttribute('src', mainProject.image);
+  mainCard.querySelector('h3').textContent = mainProject.name;
+  mainCard.querySelector('p').textContent = mainProject.description;
+  mainCard.querySelectorAll('a').forEach((anchor, k) => {
+    anchor.textContent = mainProject.technologies[k];
+  });
+
+  //
+  secProjects.forEach((projectsData, index) => {
+    const Card = document.querySelector('.secondary-projects').children[index];
+    Card.querySelector('h2').textContent = projectsData.name;
+    Card.querySelector('p').textContent = projectsData.description;
+    Card.querySelectorAll('a').forEach((anchor, i) => {
+      anchor.textContent = projectsData.technologies[i];
+    });
+  });
+  // });
```

Comment:
- [ ] please try to remove the unused commented code in order to keep your file clean and professional